src/sfizz/FilterDescription.h

```diff
@@ -15,6 +15,8 @@ namespace sfz
 struct FilterDescription
 {
     float cutoff { Default::filterCutoff };
```

Comment:
Unless I'm mistaken this is never set by an opcode. Is this on purpose?

Reply:
No worries about the delay, first of all.

Admittedly it's been quite a while since I visited this code, myself, so can't confirm or deny this particular code immediately, though knowing what I was trying to do, it was likely necessary the way I saw the code to make my workaround for CC-based cutoff modulation. Feel free to write up a more convenient alternative to this part if you need to.